src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -1043,6 +1043,10 @@ export function createApiFactoryAndRegisterActors(accessor: ServicesAccessor): I
 			},
 			onDidGrantWorkspaceTrust: (listener, thisArgs?, disposables?) => {
 				return extHostWorkspace.onDidGrantWorkspaceTrust(listener, thisArgs, disposables);
+			},
+			registerEditSessionIdentityProvider: (scheme: string, provider: vscode.EditSessionIdentityProvider) => {
+				checkProposedApiEnabled(extension, 'contribEditSessions');
```

Comment:
nit - Remove `contrib`-prefix for proposals that include source code

Reply:
I was piggybacking off the existing empty `contribEditSessions` api proposal that I added last iteration, which allows extensions to contribute a Continue Working On entry to a quickpick from core. In such cases do we recommend to create a separate api proposal rather than reusing an existing one, or to combine and rename?